### DISCUSSION TITLE
Garagen Tor Hörmann

### DISCUSSION THREAD

**Discussion by DJChrisRey** [SukramJ/aiohomematic#1338] [Ideas]
Hallo es gibt ja bei hörmann die Lüftungposition wäre cool wenn ihr das mit einbindet siehe screenshot

weil das wäre noch das i tüpfelchen!
![Bildschirmfoto 2023-12-27 um 15 54 20](https://github.com/danielperna84/hahomematic/assets/150131616/03ddc1c3-8409-4983-88c1-79571171dc3d)
Die Position ist 10% Bei lüftung.

Wäre cool so erkennt er diese auch besser. 

Weil aktuell geht ja nur öffnen - schließen

**Comment by SukramJ** [comment]
Am Besten hier mal lesen: #462

**Reply by DJChrisRey** [reply]
ah ok versteh ich leider so nicht denke mal das wird mit einem button gelöst den ich dann den Trigger in der ccu anfahren wie geht das am besten habe einen virtuellen button nun verknüpft aber wie kann ich den in ha auslösen

**Reply by SukramJ** [reply]
Du nutzt HA, also benutze HA auch. Die explizite CCU Nutzung (set_device_value) macht hier wenig Sinn, da bereits HA services den Aufruf kapseln.
Also erstelle die in HA einen Button der den Service cover.set_position aufruft, und dort einen Wert >10 und < 50 übergibt.
